pkg/sentry/server/validator/insecure/insecure.go

```diff
@@ -23,17 +23,19 @@ import (
 	"github.com/dapr/dapr/pkg/sentry/server/validator/internal"
 )
 
-type selfhosted struct{}
+// insecure implements the validator.Interface. It doesn't perform any authentication on requests.
+// It is meant to be used in self-hosted scenarios where Dapr is running on a trusted environment.
+type insecure struct{}
```

Comment:
If you have a better name pls suggest!

Reply:
LocalValidator?

Reply:
The thing is:

- it doesn’t do any validation at all. It accepts anything
- JWKS can be local too (JWKS works on both self-hosted and kubernetes)

Reply:
I think `insecure` is the best name (and definitely correct!)